### ISSUE TITLE
 Deployer\Exception\Exception::getTaskFilename(): Return value must be of type string, null returned

### ISSUE THREAD

**Issue by lptn** [deployphp/deployer#2432]
- Deployer version: v7.0.0-beta.13
- Deployment OS: macOS

1. Specify a custom task in `deploy.php` that will fail
2. run this task: `dep my-task`
3. see exception

```
Deployer\Exception\Exception::getTaskFilename(): Return value must be of type string, null returned


#0 /Users/me/code/project/vendor/deployer/deployer/src/Executor/Messenger.php(82): Deployer\Exception\Exception->getTaskFilename()
#1 /Users/me/code/project/vendor/deployer/deployer/src/Executor/Worker.php(43): Deployer\Executor\Messenger->renderException(Object(Deployer\Exception\RunException), Object(Deployer\Host\Host))
#2 /Users/me/code/project/vendor/deployer/deployer/src/Command/WorkerCommand.php(58): Deployer\Executor\Worker->execute(Object(Deployer\Task\Task), Object(Deployer\Host\Host))
#3 /Users/me/code/project/vendor/symfony/console/Command/Command.php(256): Deployer\Command\WorkerCommand->execute(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#4 /Users/me/code/project/vendor/symfony/console/Application.php(971): Symfony\Component\Console\Command\Command->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#5 /Users/me/code/project/vendor/symfony/console/Application.php(290): Symfony\Component\Console\Application->doRunCommand(Object(Deployer\Command\WorkerCommand), Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#6 /Users/me/code/project/vendor/symfony/console/Application.php(166): Symfony\Component\Console\Application->doRun(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#7 /Users/me/code/project/vendor/deployer/deployer/src/Deployer.php(289): Symfony\Component\Console\Application->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#8 /Users/me/.composer/vendor/deployer/deployer/bin/dep(105): Deployer\Deployer::run('7.0.0-beta.13', '...')
```
_____________
**UPD**: important note, it happens when I use `artisan()` helper in my task:
```php
task('artisan:server:audit', artisan('server:audit', ['showOutput', 'runInCurrent']));
```

this Laravel’s console command should exit with non-zero error code.

stacktrace in this case doesn't include info about deployer.php file
```json
[
    {
        "file": "/Users/me/code/project/vendor/deployer/deployer/src/Exception/RunException.php",
        "line": 35,
        "function": "__construct",
        "class": "Deployer\\Exception\\Exception",
        "object": {},
        "type": "->",
        "args": [
            "The command \"exit 1\" failed.",
            1
        ]
    },
    {
        "file": "/Users/me/code/project/vendor/deployer/deployer/src/Component/ProcessRunner/ProcessRunner.php",
        "line": 76,
        "function": "__construct",
        "class": "Deployer\\Exception\\RunException",
        "object": {},
        "type": "->",
        "args": [
            {},
            "exit 1",
            1,
            "",
            ""
        ]
    },
{
        "file": "/Users/me/code/project/vendor/deployer/deployer/src/functions.php",
        "line": 382,
        "function": "Deployer\\{closure}",
        "args": [
            "/usr/local/bin/php ~/localhost/current/artisan server:audit",
            []
        ]
    },
{
        "file": "/Users/me/code/project/vendor/deployer/deployer/recipe/laravel.php",
        "line": 67,
        "function": "Deployer\\run",
        "args": [
            "{{bin/php}} {{current_path}}/artisan server:audit"
        ]
    },
{
        "function": "Deployer\\{closure}",
        "args": []
    },
    {
        "file": "/Users/me/code/project/vendor/deployer/deployer/src/Task/Task.php",
        "line": 43,
        "function": "call_user_func",
        "args": [
            {}
        ]
    },
...
```

**Comment by Schrank** [comment]
Hi @lptn and thanks for your report!

Can you give me an example what you mean by "failing task"?

**Comment by lptn** [comment]
Hi @Schrank 
I'm still debugging it and just updated my initial message with some important info

**Comment by Schrank** [comment]
Too fast :D Ok, I'll have a look later :)

**Comment by antonmedv** [comment]
Fixed in 1851e1812a2b74f78a109d0d2b19fb7c5ea35a48.
